src/Component/Navbar/Navbar.css

```diff
@@ -2,7 +2,7 @@ h3 {
   font-size: 6rem;
 }
 .navbar {
-  background-color: aqua;
+  background-color: rgb(98, 255, 0);
```

Comment:
please do not change color its not looking good

Reply:
> please do not change color its not looking good

ok done